<?xml version="1.0"?>
<launch>
    	<group ns="left_group">
		<include file="$(find baxter_imp)/launch/demo.launch" />
		<node name="motion_planner_node" pkg="baxter_imp" type="motion_planner.py" args="--limb left" output="screen"  />
	</group>
</launch>
